Use the standard nullptr type in installed V8 development headers.
Addon compilers need not provide the global nullptr_t extension available
with the bundled libc++. These equivalent qualifications preserve the ABI.

--- a/include/node/v8-object.h
+++ b/include/node/v8-object.h
@@ -4,6 +4,8 @@
 
 #ifndef INCLUDE_V8_OBJECT_H_
 #define INCLUDE_V8_OBJECT_H_
+
+#include <cstddef>
 
 #include "cppgc/garbage-collected.h"
 #include "cppgc/name-provider.h"
@@ -405,7 +407,7 @@
   // is removed.
   V8_WARN_UNUSED_RESULT Maybe<bool> SetNativeDataProperty(
       Local<Context> context, Local<Name> name,
-      AccessorNameGetterCallback getter, nullptr_t setter = nullptr,
+      AccessorNameGetterCallback getter, std::nullptr_t setter = nullptr,
       Local<Value> data = Local<Value>(), PropertyAttribute attributes = None,
       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
       SideEffectType setter_side_effect_type = SideEffectType::kHasSideEffect) {
--- a/include/node/v8-template.h
+++ b/include/node/v8-template.h
@@ -117,7 +117,7 @@
   // is removed.
   void SetNativeDataProperty(
       Local<Name> name, AccessorNameGetterCallback getter,
-      nullptr_t setter = nullptr, Local<Value> data = Local<Value>(),
+      std::nullptr_t setter = nullptr, Local<Value> data = Local<Value>(),
       PropertyAttribute attribute = None,
       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
       SideEffectType setter_side_effect_type = SideEffectType::kHasSideEffect) {
@@ -748,7 +748,7 @@
             PropertyHandlerFlags::kInternalNewCallbacksSignatures));
   }
 
-  static NamedPropertySetterCallbackV2 ConvertSetter(nullptr_t) {
+  static NamedPropertySetterCallbackV2 ConvertSetter(std::nullptr_t) {
     return nullptr;
   }
   static NamedPropertySetterCallbackV2 ConvertSetter(
@@ -760,7 +760,7 @@
     return NamedPropertySetterCallbackV2(value);
   }
 
-  static NamedPropertyDefinerCallbackV2 ConvertDefiner(nullptr_t) {
+  static NamedPropertyDefinerCallbackV2 ConvertDefiner(std::nullptr_t) {
     return nullptr;
   }
   static NamedPropertyDefinerCallbackV2 ConvertDefiner(
@@ -775,11 +775,11 @@
  public:
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // NamedPropertySetterCallbackV2/NamedPropertyDefinerCallbackV2 is done.
-  template <typename TSetter = nullptr_t, typename TDefiner = nullptr_t>
-    requires((std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t, typename TDefiner = std::nullptr_t>
+    requires((std::is_same_v<TSetter, std::nullptr_t> ||
               std::is_same_v<TSetter, NamedPropertySetterCallback> ||
               std::is_same_v<TSetter, NamedPropertySetterCallbackV2>) &&
-             (std::is_same_v<TDefiner, nullptr_t> ||
+             (std::is_same_v<TDefiner, std::nullptr_t> ||
               std::is_same_v<TDefiner, NamedPropertyDefinerCallback> ||
               std::is_same_v<TDefiner, NamedPropertyDefinerCallbackV2>))
   NamedPropertyHandlerConfiguration(
@@ -804,8 +804,8 @@
 
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // NamedPropertySetterCallbackV2/NamedPropertyDefinerCallbackV2 is done.
-  template <typename TSetter = nullptr_t>
-    requires(std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t>
+    requires(std::is_same_v<TSetter, std::nullptr_t> ||
              std::is_same_v<TSetter, NamedPropertySetterCallback> ||
              std::is_same_v<TSetter, NamedPropertySetterCallbackV2>)
   explicit NamedPropertyHandlerConfiguration(
@@ -828,11 +828,11 @@
 
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // NamedPropertySetterCallbackV2/NamedPropertyDefinerCallbackV2 is done.
-  template <typename TSetter = nullptr_t, typename TDefiner = nullptr_t>
-    requires((std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t, typename TDefiner = std::nullptr_t>
+    requires((std::is_same_v<TSetter, std::nullptr_t> ||
               std::is_same_v<TSetter, NamedPropertySetterCallback> ||
               std::is_same_v<TSetter, NamedPropertySetterCallbackV2>) &&
-             (std::is_same_v<TDefiner, nullptr_t> ||
+             (std::is_same_v<TDefiner, std::nullptr_t> ||
               std::is_same_v<TDefiner, NamedPropertyDefinerCallback> ||
               std::is_same_v<TDefiner, NamedPropertyDefinerCallbackV2>))
   NamedPropertyHandlerConfiguration(
@@ -875,7 +875,7 @@
             PropertyHandlerFlags::kInternalNewCallbacksSignatures));
   }
 
-  static IndexedPropertySetterCallback ConvertSetter(nullptr_t) {
+  static IndexedPropertySetterCallback ConvertSetter(std::nullptr_t) {
     return nullptr;
   }
   static IndexedPropertySetterCallback ConvertSetter(
@@ -887,7 +887,7 @@
     return IndexedPropertySetterCallback(value);
   }
 
-  static IndexedPropertyDefinerCallback ConvertDefiner(nullptr_t) {
+  static IndexedPropertyDefinerCallback ConvertDefiner(std::nullptr_t) {
     return nullptr;
   }
   static IndexedPropertyDefinerCallback ConvertDefiner(
@@ -902,11 +902,11 @@
  public:
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // IndexedPropertySetterCallback/IndexedPropertyDefinerCallback is done.
-  template <typename TSetter = nullptr_t, typename TDefiner = nullptr_t>
-    requires((std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t, typename TDefiner = std::nullptr_t>
+    requires((std::is_same_v<TSetter, std::nullptr_t> ||
               std::is_same_v<TSetter, IndexedPropertySetterCallback> ||
               std::is_same_v<TSetter, IndexedPropertySetterCallbackV2>) &&
-             (std::is_same_v<TDefiner, nullptr_t> ||
+             (std::is_same_v<TDefiner, std::nullptr_t> ||
               std::is_same_v<TDefiner, IndexedPropertyDefinerCallback> ||
               std::is_same_v<TDefiner, IndexedPropertyDefinerCallbackV2>))
   IndexedPropertyHandlerConfiguration(
@@ -931,8 +931,8 @@
 
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // IndexedPropertySetterCallback/IndexedPropertyDefinerCallback is done.
-  template <typename TSetter = nullptr_t>
-    requires(std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t>
+    requires(std::is_same_v<TSetter, std::nullptr_t> ||
              std::is_same_v<TSetter, IndexedPropertySetterCallback> ||
              std::is_same_v<TSetter, IndexedPropertySetterCallbackV2>)
   explicit IndexedPropertyHandlerConfiguration(
@@ -955,11 +955,11 @@
 
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // IndexedPropertySetterCallback/IndexedPropertyDefinerCallback is done.
-  template <typename TSetter = nullptr_t, typename TDefiner = nullptr_t>
-    requires((std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t, typename TDefiner = std::nullptr_t>
+    requires((std::is_same_v<TSetter, std::nullptr_t> ||
               std::is_same_v<TSetter, IndexedPropertySetterCallback> ||
               std::is_same_v<TSetter, IndexedPropertySetterCallbackV2>) &&
-             (std::is_same_v<TDefiner, nullptr_t> ||
+             (std::is_same_v<TDefiner, std::nullptr_t> ||
               std::is_same_v<TDefiner, IndexedPropertyDefinerCallback> ||
               std::is_same_v<TDefiner, IndexedPropertyDefinerCallbackV2>))
   IndexedPropertyHandlerConfiguration(
@@ -983,11 +983,11 @@
 
   // TODO(https://crbug.com/348660658): cleanup once migration to
   // IndexedPropertySetterCallback/IndexedPropertyDefinerCallback is done.
-  template <typename TSetter = nullptr_t, typename TDefiner = nullptr_t>
-    requires((std::is_same_v<TSetter, nullptr_t> ||
+  template <typename TSetter = std::nullptr_t, typename TDefiner = std::nullptr_t>
+    requires((std::is_same_v<TSetter, std::nullptr_t> ||
               std::is_same_v<TSetter, IndexedPropertySetterCallback> ||
               std::is_same_v<TSetter, IndexedPropertySetterCallbackV2>) &&
-             (std::is_same_v<TDefiner, nullptr_t> ||
+             (std::is_same_v<TDefiner, std::nullptr_t> ||
               std::is_same_v<TDefiner, IndexedPropertyDefinerCallback> ||
               std::is_same_v<TDefiner, IndexedPropertyDefinerCallbackV2>))
   IndexedPropertyHandlerConfiguration(
